### ISSUE TITLE
Bug: IE11 with Suspense doesn't support direct child with inline display: flex style

### ISSUE THREAD

**Issue by AdrianoRuberto** [facebook/react#18906]
If the direct children of `<React.Suspense />` have an inline `display: flex` style, when `Suspense` will be resolved it will NOT remove `display: none !important`, thus there will be a blank page.

This bug is only on the last version of IE11 and only with inline style of `display: flex`.

React version: 16.13.1

## Steps To Reproduce

1. Add <React.Suspense /> 
2. Add a direct child with an inline style as `display: flex`
3. Lazy load something

Link to code example: https://github.com/AdrianoRuberto/ie11-suspense-bug

## The current behavior
A blank page with IE11

## The expected behavior
The loaded page.

## Workarounds
Basically, remove the inline style with `display: flex` to be the direct child of `<React.Suspense />` does the trick, such as:
1. Add a div between the `React.Suspense` and the `<div style={{display: "flex"}}>`
2. Use a `className` and move the inline style inside the stylesheet.

**Comment by gaearon** [comment]
Any hunch as to why IE behavior is different?

**Comment by AdrianoRuberto** [comment]
No, but with an older version (I tried with version 11.0.9600), it work like a charm.

It's not the first time that IE11 make me frown.

Maybe the inline style doesn't work the same with IE11 than the others browsers ?

**Comment by amir-beheshty** [comment]
I'm experiencing this exact issue.

**Comment by gaearon** [comment]
We've dropped IE from supported browsers in React 18 so this is unlikely to be fixed.

Sorry.
